os/test/src-jvm/ZipOpJvmTests.scala

```diff
@@ -36,19 +36,22 @@ object ZipOpJvmTests extends TestSuite {
         dest = wd / "unzipped folder"
       )
 
-      val paths = os.walk(unzippedFolder)
-      val expected = Seq(
+      val paths = os.walk(unzippedFolder).toList.sorted
+      val expected = List(
         // Files get included in the zip root using their name
         wd / "unzipped folder/File.txt",
         wd / "unzipped folder/Multi Line.txt",
         // Folder contents get included relative to the source root
-        wd / "unzipped folder/nestedA",
-        wd / "unzipped folder/nestedB",
-        wd / "unzipped folder/one.txt",
-        wd / "unzipped folder/nestedA/a.txt",
-        wd / "unzipped folder/nestedB/b.txt"
-      )
-      assert(paths.sorted == expected)
```

Comment:
@lihaoyi I'm not sure why the expected output should be this, since the file tree likes following
```
.
├── File.txt
├── Multi Line.txt
├── emptyFolder
├── folder1
│   └── one.txt
├── folder2
│   ├── nestedA
│   │   └── a.txt
│   └── nestedB
│       └── b.txt

```

it seems missing the folder `folder1` and `folder2` level ?